packages/terra-clinical-site/src/examples/action-header/Index.jsx

```diff
@@ -20,7 +20,7 @@ const HeaderExamples = () => (
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
     <h2>Action Header</h2>
```

Comment:
lets remove `<h2>Action Header</h2>` as the title is now included in the props-table